packages/global/components/site-footer.marko

```diff
@@ -16,7 +16,7 @@ $ const tagline = site.get("tagline");
   modifiers=input.modifiers
   attrs=input.attrs
 >
-  <default-theme-site-footer-container block-name=blockName modifiers=["secondary"]>
+  <default-theme-site-footer-container block-name=blockName modifiers=["secondary"] attrs={ "aria-label": "Secondary Footer" }>
```

Comment:
https://github.com/parameter1/base-cms/pull/303/commits/0b569d64103216048d06ca42b55e7928db1595a0

This requires the changes made here: https://github.com/parameter1/base-cms/pull/303 which requires a dependency upgrade.